### PR TITLE
Added start-account-storage to process iframe messages

### PR DESCRIPTION
**Pull request by rchild-okta**
- Responds to messages to add, list, and delete new accounts
- Supports multiple trusted origins
- Supports multiple accounts per origin
- Caps number of cached accounts to 5 (FIFO)
- Updated sample app to use new message format
- Tests + Linting

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -42,6 +44,10 @@
             $('#account-display-name').val('');
           });
 
+          $('#delete-account').click(function() {
+            deleteAccount();
+          });
```
Comment (ukilon-okta) [inline index.html:49]: nit: can this be `$('#delete-account').click(deleteAccount);`?

Comment (rchild-okta) [inline index.html:49]: Heh, yeah, but I didn't want to modify the style/structure of this file too much (most of the focus is on the account-storage logic)

**Comment by ukilon-okta** [comment]
🚢

**Comment by rchild-okta** [comment]
\+ @okta/rex-team, mostly around [line 106](https://github.com/mcguinness/account-chooser-sample/pull/1/files#diff-a8ca9ebd92a5074d514fb301e0afa50bR106).

Summary: We're going to be hosting an iframe with the account-storage logic on login.okta.com, and it will respond to incoming requests to store and retrieve account information via [postMessage](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage).

Main concern is reviewing [this function](https://github.com/mcguinness/account-chooser-sample/pull/1/files#diff-82dd2faba4e403441fcb80d64f118806R173) to make sure we're not responding to origins we're not anticipating.

**Pull request merged**